driver/sdk.md

```diff
@@ -1,39 +1,43 @@
-
 # Babelfish SDK
 
 The [Babelfish SDK](https://github.com/bblfsh/sdk/) contains tools and libraries
-required to create a Babelfish driver.
+required to create a Babelfish driver and some APIs to analyze or transform
+the generated UAST.
+
+## Preparations
 
-## Prerequisites
+### Dependencies
 
-Babelfish has the following prerequisites:
+The Babelfish SDK has the following dependencies:
 
 * [Docker](https://www.docker.com/get-docker)
 * [Go](https://golang.org/dl/)
 
-## Installing
+Make sure that you've correctly set your GOROOT and GOPATH environment
+variables. 
 
-Install the Babelfish SDK:
+### Installing
+
+Install the Babelfish SDK. Native drivers and the external tools can be developed
+on any programming language but the non-native driver is developed using Go so you
```

Comment:
- developed *in* any language.
- non-native driver is developed using Go -> normalizer is written in Go

I find this whole paragraph confusing though. The fact that the SDK is written in go is the one that determines that you need to use go to install it, no the fact that the normalizer will have to be written in Go. I'd remove it.